puppet/modules/slave/manifests/init.pp

```diff
@@ -162,6 +162,30 @@
     }
   }
 
+  # Needed by foreman_openscap gem dependency OpenSCAP
+  if $osfamily == 'RedHat' {
+    $repo = $::operatingsystem ? {
+        'Fedora' => 'fedora',
+        default  => 'epel',
+      }
+
+      if $::operatingsystemmajrelease == undef {
+        $versions_array = split($::operatingsystemrelease, '\.') # facter 1.6
+        $major = $versions_array[0]
+      } else {
+        $major = $::operatingsystemmajrelease # facter 1.7+
+      }
+
+      yumrepo { 'isimluk-openscap':
+        enabled  => 1,
+        gpgcheck => 0,
+        baseurl  => "http://copr-be.cloud.fedoraproject.org/results/isimluk/OpenSCAP/${repo}-${major}-\$basearch/",
```

Comment:
could you also set `includepkgs => ['openscap']` so we don't pick up other stuff from this repo?